src/yacfg/config_data.py

```diff
@@ -1,66 +1,48 @@
-# Copyright 2018 Red Hat Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
 import datetime
 import time
+from typing import Dict
 from collections import namedtuple
 
 from . import __version__
 from .meta import NAME
 
+RenderOptions = namedtuple("RenderOptions", ["generator_notice", "licenses"])
+
 
-def add_template_metadata(config_data):
-    """add templating metadata to original template data
+def add_template_metadata(config_data: Dict) -> None:
+    """Add templating metadata to the original template data.
 
-    :param config_data:  original template data
+    :param config_data: Original template data.
     :type config_data: dict
     """
     now = datetime.datetime.now()
+    date_format = "%Y-%m-%d"
+    time_format = "%H:%M"
 
     config_data["metadata"] = {
         "tool_name": NAME,
         "tool_version": __version__,
         "datetime": {
-            "datetime": now.strftime("%Y-%m-%d %H:%M"),
+            "datetime": now.strftime(f"{date_format} {time_format}"),
             "year": now.strftime("%Y"),
-            "time": now.strftime("%H:%M"),
-            "date": now.strftime("%Y-%m-%d"),
+            "time": now.strftime(time_format),
+            "date": now.strftime(date_format),
             "unix": time.time(),
         },
     }
 
 
-def add_render_config(config_data, render_options):
-    """add render related config to original template data
+def add_render_config(config_data: Dict, render_options: "RenderOptions") -> None:
```

Comment:
`from __future__ import annotations` should be available in 3.7+, so that could be used, to avoid the quotes. (https://docs.python.org/3/library/__future__.html)

Reply:
Hey, you're absolutely right! I actually overlooked quite a few things. Thanks for being an extra set of eyes! I appreciate it!